test/man/checkManPages

```diff
@@ -181,18 +194,22 @@ def _get_flags(s):
     # Look for all comma separated flags
     ls = ls.strip().split(',')
     for f in ls:
-        sf = f.strip();
+        sf = f.strip()
         try:
             if sf[0] == '-':
                 # Add flag to the return string
                 rs += sf.split()[0]+','
+            elif sf[0] == '\\':
+                rs += sf.split()[0]+','
             else:
                 break
         except IndexError:
             continue
+    if rs[2] == ",":
+        rs = rs[0:3] + " " + rs[3:len(rs)]
     if rs:
         # Remove any trailing text
-        return rs.strip().split()[0]
+        return rs.strip()
```

Comment:
I'm not guessing what led to this change?

Reply:
As I have changed the logic behind the function `_get_flags` , we did'nt fall under the condition as we did'nt get any trailing text fromt the flags.